nuxt.config.ts

```diff
@@ -1,5 +1,13 @@
 import { fileURLToPath } from "node:url";
 
+let nuxtdevtools_installed;
```

Comment:
why is this necessary?

Reply:
Well it isn't but `pnpm prune --prod` does not work without it and that is an integral part of the standardized process.
Try to remove the code and do the following manually:
* `pnpm install`
* `pnpm run build`
* `pnpm prune --prod`

pnpm prune removes about 250 packages form node_modules that don't have much business in the container on the production instance. Among them the nuxt dev tools.